operatorconfig/driverconfig/powerflex/v2.6.0/node.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ServiceAccount
```

Comment:
I think we are missing the new env variables introduced in 2.6:
X_CSI_APPROVE_SDC_ENABLED
X_CSI_RENAME_SDC_ENABLED
X_CSI_RENAME_SDC_PREFIX

Reply:
same comment as above